test/components/DayPickerRangeController_spec.jsx

```diff
@@ -696,6 +696,31 @@ describe('DayPickerRangeController', () => {
               expect(isSameDay(minimumNightsCalls[0].args[2], minimumNightsEndSpan)).to.equal(true);
             });
           });
+
+          describe('minimumNights changed', () => {
+            it('calls deleteModifierFromRange with start date + old min nights, and `blocked-minimum-nights`', () => {
+              const deleteModifierFromRangeSpy = sinon.spy(DayPickerRangeController.prototype, 'deleteModifierFromRange');
+              const startDate = today;
+              const focusedInput = START_DATE;
+              const minimumNights = 5;
+              const wrapper = shallow(<DayPickerRangeController
```

Comment:
fwiw, the linter will soon require:
```jsx
const wrapper = shallow((
  <DayPickerRangeController
    startDate={startDate}
    focusedInput={focusedInput}
    minimumNights={minimumNights}
  />
));
```

Reply:
we'll have to do a wholesale change of... most tests at that point. 😬